website/docs/cloud-docs/api-docs/run-tasks/run-tasks.mdx

```diff
@@ -75,7 +75,7 @@ Properties without a default value are required unless otherwise specified.
 | `data.type`                   | string |         | Must be `"tasks"`.                                                                            |
 | `data.attributes.name`        | string |         | The name of the task. Can include letters, numbers, `-`, and `_`.                             |
 | `data.attributes.url`         | string |         | URL to send a run task payload.                                                               |
-| `data.attributes.description` | string |         | The description of the run task. Can include spaces, letters, numbers, and special characters |
+| `data.attributes.description` | string |         | The description of the run task. Can include spaces, letters, numbers, and special characters up to 300 characters. |
```

Comment:
```suggestion
| `data.attributes.description` | string |         | The description of the run task. Can be up to 300 characters long including spaces, letters, numbers, and special characters. |
```

Suggest changing the character description to "Can be up to 300 characters long including spaces, letters, numbers, and special characters" for clarity.